uwblueprint/humane-society
# S24/jerry/revert roles table

## Notion ticket link

[Notion Ticket](https://www.notion.so/uwblueprintexecs/Add-Status-Column-to-User-Table-Change-Role-to-Enum-093f2ea5b2074a30b6d68758884f530b?pvs=4)


## Implementation description
* Removed roles table, and reverted all user related code to use roles enum
* Added status column in users table (required field)



## Steps to test
1. Run `node migrate down --to 0` in backend container
2. Run `node migrate up` in backend container
3. See if role and status columns are in users table



## What should reviewers focus on?
* Make sure migrations work


## Checklist
